src/components/GridTitle.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Link from "gatsby-link";
+import withStyles from "elevate-ui/withStyles";
+import Icon from "elevate-ui-icons/Icon";
+
+const GridCard = ({ classes, name, icon, color }) => {
+  return (
+    <div className={classes.root}>
+      <div className={classes.title}>
+        <div className={classes.icon}>
+          <Icon size={24} color={color} name={icon} />
+        </div>
+        <div style={{ color }} className={classes.name}>
```

Comment:
Looks like this div gets all its styling from `title`, could we move container-related styling up to `root` and text-related styling down to `name`?